spec/support/helpers.rb

```diff
@@ -3,14 +3,25 @@
 
 module Spec
   module Helpers
-    def self.bundler_version
-      return @bundler_version if defined?(@bundler_version)
-      versions = Bundler.with_clean_env do
+    def self.bundler_versions
+      @bundler_versions ||= Bundler.with_clean_env do
         `gem list bundler`.lines.grep(/^bundler /).first.scan(/\d+\.\d+\.\d+/)
       end
+    end
+
+    def self.bundler_version
+      return @bundler_version if defined?(@bundler_version)
+
+      versions = bundler_versions
+
       to_find = ENV["TEST_BUNDLER_VERSION"] || ENV["BUNDLER_VERSION"]
-      @bundler_version = versions.detect { |v| v.include?(to_find) }
-      raise "Unable to find bundler version: #{to_find.inspect}" if @bundler_version.nil?
+      if to_find
+        @bundler_version = versions.detect { |v| v.match?(/\b#{Regexp.escape(to_find)}\b/) }
```

Comment:
Instead of going the regex route, can you just swapped over to `start_with?`.

side effect: I `versions.detect {|v| v.start_with?('') }` == `versions.first`

Separating these out into another method work well. Does feel like a 1 liner though (spread across 3 lines)

Reply:
Oh yeah...start_with? makes more sense.